pkg/services/dashboards/database/database.go

```diff
@@ -710,6 +711,40 @@ func (d *DashboardStore) deleteDashboard(cmd *dashboards.DeleteDashboardCommand,
 		"DELETE FROM dashboard_acl WHERE dashboard_id = ?",
 	}
 
+	if d.features.IsEnabled(featuremgmt.FlagPublicDashboardsEmailSharing) {
```

Comment:
feature flag exists in the OSS codebase, but requires valid enterprise license to be enabled